packages/utils/upgrade/src/__tests__/core/logger.test.ts

```diff
@@ -0,0 +1,121 @@
+import { createLogger } from '../../core';
+
+import type { Logger } from '../../core';
+import chalk from 'chalk';
+
+const noop = () => {};
+
+describe('Logger', () => {
+  const text = 'Hello World!';
+  const now = new Date();
+
+  beforeAll(() => {
+    jest.useFakeTimers({ now });
+  });
+
+  afterAll(() => {
+    jest.useRealTimers();
+  });
+
+  beforeEach(() => {
+    jest.spyOn(console, 'log').mockImplementation(noop);
```

Comment:
we should really switch to using something like jest-mock-console throughout most of our tests but that's not your problem to solve unless you want it to be 😆